### PR TITLE
Add a `AuFileCard` component

### LINKED ISSUE #213: `AuFileCard` component
Webuniversum provides a `VoMuFileCard` component which looks like this:

image

I've modernized this in the Loket app, but that's not the only app that uses this, so I've added it to a shared addon. It might be better to add it to ember-appuniversum so it can be reused by other projects if they need it?

### PR DESCRIPTION
**Pull request by Windvis**
Based on the [custom vo-mu-filecard component in Loket](https://github.com/lblod/frontend-loket/blob/2a514e74f9b509a472bdc6033a37505fd0fde93f/app/templates/components/vo-mu-file-card.hbs).

image

Closes #213

### CODE REVIEW

**Comment by Windvis** [comment]
During a code review where I added a similar component, @aatauil remarked that it might be better to have an explicit flag to show the delete button instead of deriving this from the handler state. I didn't add it since it seemed unnecessary, but the current version might not be ideal if we want to conditionally show the delete button.

Current version:
```hbs
<AuFileCard 
  @filename={{file.filename}}
  @fileSize={{file.humanReadableSize}}
  @downloadLink={{file.downloadLink}}
  @onDelete={{if @someCondition (fn this.deleteFile file)}} 
/>
```

With the extra flag:
```hbs
<AuFileCard 
  @filename={{file.filename}}
  @fileSize={{file.humanReadableSize}}
  @downloadLink={{file.downloadLink}}
  @isDeletable={{@someCondition}}
  @onDelete={{fn this.deleteFile file}} 
/>
```
I personally don't like having to set 2 arguments when the same can be accomplished with 1, but readability of code is also important.

Any opinions about this? The webuniversum component does have a separate `active` argument, but I found that name very confusing :smile:.

File: addon/components/au-file-card.hbs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<article class="au-file-card au-o-box au-o-box--small" ...attributes>
+  {{#if this.isRemovable}}
+    <button
```
[the remark this answers is not in the document]
Comment (Dietr) [reply addon/components/au-file-card.hbs:3]: Yes, using a button would be cleaner. We can probably define an icon-only skin for the button component.

File: addon/components/au-file-card.hbs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <span
+        class="au-file-card__file-size"
+        data-test-file-card-file-size
+      >({{@fileSize}})</span>
+    {{/if}}
+  </AuHeading>
+
+  {{#if this.isDownloadable}}
+    <div class="au-u-margin-top-tiny">
+      <a
```
[the remark this answers is not in the document]
Comment (Dietr) [reply addon/components/au-file-card.hbs:27]: Maybe we can add an option on the AuLinkExternal component to set the target?

**Comment by Dietr** [comment]
Looks ok for me. I'm missing the documentation part though so was not really able to test this.

**Comment by Dietr** [comment]
@Windvis My bad, it is indeed. For some reason I expected this to be in the forms section next to the file upload. But in the content section is fine.

**Review by Dietr — APPROVED** [review]

**Pull request merged**